project.php

```diff
@@ -1962,6 +1951,25 @@ function do_smooth_reading()
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
+function sr_echo_time_form($min_days, $max_days, $default_days, $label, $extend = false)
```

Comment:
I think the function arguments make a little more sense if you put the label first.
Also add a comment about the expected form of $label, eg:
```
$label is an sprintf formatting string with three placement values:
* %1$d will be replaced with a form field for number-of-days input
* %2$d is the minimum number of days available to the form
* %3$d is the maximum number of days available to the form
```

Reply:
Done this